diff --git a/src/alejandra/src/builder.rs b/src/alejandra/src/builder.rs
index 3eb4f0c..ccac4c8 100644
--- a/src/alejandra/src/builder.rs
+++ b/src/alejandra/src/builder.rs
@@ -61,7 +61,7 @@ fn build_step(
                         format!(
                             "{0:<1$}{2}",
                             "",
-                            2 * build_ctx.indentation,
+                            4 * build_ctx.indentation,
                             line,
                         )
                     }
@@ -96,7 +96,7 @@ fn build_step(
                 add_token(
                     builder,
                     rnix::SyntaxKind::TOKEN_WHITESPACE,
-                    &format!("{0:<1$}", "", 2 * build_ctx.indentation),
+                    &format!("{0:<1$}", "", 4 * build_ctx.indentation),
                 );
             }
         }
